### PR TITLE
Users should only submit from local scratch

### PR DESCRIPTION
**Pull request by brianhlin**
I've only gone through my exercises + Derek's to make sure we're telling user to create their exercise dirs on local scratch on the training host

### CODE REVIEW

**Review by djw8605 — APPROVED** [review]
Seems ok.  But I suspect there needs to be more instructions to copy data around in day 4.

File: docs/materials/day4/part1-ex3-blast-split.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Setup
 -----
 
 -   Make sure you are still logged into `training.osgconnect.net`
--   Make sure you are in the directory named `thur-blast-data` under the `stash` filesystem.
+-   Make sure you are in the directory named `thur-blast-data` under your local scratch directory
```
Comment (ChristinaLK) [inline docs/materials/day4/part1-ex3-blast-split.md:15]: how big is the data? if the individual files are big, it should go into stash. if they're small, local scratch is the right place...

**Review by ChristinaLK — APPROVED** [review]
I left one comment (which I think is not an issue, but just wanted to check) - otherwise this looks good.

File: docs/materials/day4/part1-ex3-blast-split.md (hunk 1 above)
Comment (brianhlin) [reply docs/materials/day4/part1-ex3-blast-split.md:15]: The original test data is about 100MB but in this exercise we're splitting it into 100 1MB files to act as the inputs for the jobs.

**Pull request merged**